src/satellite_images_nso/api/nso_georegion.py

```diff
@@ -48,12 +73,15 @@ class nso_georegion:
 
     def __init__(
         self,
-        path_to_geojson: str,
         output_folder: str,
         username: str,
         password: str,
+        path_to_geojson: str = None,
+        coordinates: str = None,
+        previous_path_to_geojson: str = None,
+        previous_link: str = None,
         cloud_detection_model_path: str = None,
-    ):
+    ) -> None:
```

Comment:
You can do `-> None`, but it is considered superfluous.

Reply:
fixed